upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.16.yaml.template

```
@@ -3893,6 +3893,9 @@ spec:
             # Enable Prometheus process metrics collection
             - name: FELIX_PROMETHEUSPROCESSMETRICSENABLED
               value: "{{- or .Networking.Calico.PrometheusProcessMetricsEnabled "true" }}"
+            # Enable / Disable source/destination checks in aws
+            - name: FELIX_AWSSRCDSTCHECK
+              value: "{{- or .Networking.Calico.AwsSrcDstCheck "DoNothing" -}}"
```

This way CrossSubnet will works same as it does now, but using the new setting.
```suggestion
            # Enable / Disable source/destination checks in AWS
            - name: FELIX_AWSSRCDSTCHECK
              value: "{{- if and (eq .CloudProvider "aws") (.Networking.Calico.CrossSubnet) -}}Disable{{- else -}} {{- or .Networking.Calico.AwsSrcDstCheck "DoNothing" -}} {{- end -}}"
```